src/Microsoft.PowerShell.Commands.Utility/commands/utility/ConvertFrom-SddlString.cs

```diff
@@ -0,0 +1,261 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+#if !UNIX
+
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Management.Automation;
+using System.Security.AccessControl;
+using System.Security.Principal;
+using System.Text;
+using System.ComponentModel;
+
+namespace Microsoft.PowerShell.Commands
+{
+    /// <summary>
+    /// Converts a SDDL string into an object-based representation of a security descriptor.
+    /// </summary>
+    [Cmdlet(VerbsData.ConvertFrom, "SddlString", HelpUri = "https://go.microsoft.com/fwlink/?LinkId=623636", RemotingCapability = RemotingCapability.None)]
+    [OutputType(typeof(SecurityDescriptorInfo))]
+    public sealed class ConvertFromSddlStringCommand : PSCmdlet
```

Comment:
I'd move the commit in separate PR  - previously we had PR #3936 to make this conversion but I had many questions and that PR was not merged.

Reply:
@daxian-dbw I agree. Please open tracking issue after the PR will merged.

Reply:
Issue #8365 was opened for this.